tests/test_flow/test_persistence_fuzz.py

```diff
@@ -386,10 +385,8 @@ def _expected_called_names(
 
 
 @pytest.fixture(scope="function")
-def fuzzer(builder, make_list, tmp_path):
-    fake_cloud_store = FakeCloudStore(str(tmp_path / "BNTESTDATA_FAKE_CLOUD"))
-    builder.set("core__persistent_cache__cloud_store", fake_cloud_store)
-    return Fuzzer(builder, make_list)
+def fuzzer(fake_gcs_builder, make_list, tmp_path):
```

Comment:
Instead of doing this, what do you think about adding a marker that basically says don't run with real gcs? If that marker is present for a test, then we skip if the test also has `real-gcp` marker.

Reply:
I thought about this but ended up leaning against it. It doesn't seem to be possible to directly attach a mark to a fixture, so we'd have to do it by parametrizing the fixture (with a single parameter value). I don't love the proliferation of extra parameters, and of extra test instances that get created and then filtered out. I also didn't love the idea of adding more cases to our top-level filtering to solve this problem. That said, I'm not crazy about this solution either and I could probably be persuaded either way.

BTW, it's interesting to note that some of the limitations in pytest's parametrization are similar to those of Bionic's multiplicity. For example, in both frameworks it's frustrating that you can't dynamically set the number of parameters/cases based on the value of another fixture/entity. I'm really looking forward to solving this in Bionic (with descriptors).